### PR TITLE
add faktory redis timeout env var

### PR DESCRIPTION
**Pull request by phawzy**

### CODE REVIEW

File: storage/redis.go [hunk 1]
```diff
@@ -131,8 +131,11 @@ func BootRedis(path string, sock string) (func(), error) {
 			}
 		}()
 	}
-
-	secs := 60
+        secs_env, isset := os.LookupEnv("FAKTORY_REDIS_TIMEOUT")
+	secs := 120
+	if isset {
+		secs, err = strconv.Atoi(secs_env)
```
Comment (yakout) [inline storage/redis.go:137]: check for the err

File: storage/redis.go [hunk 2]
```diff
@@ -131,8 +131,11 @@ func BootRedis(path string, sock string) (func(), error) {
 			}
 		}()
 	}
-
-	secs := 60
+        secs_env, isset := os.LookupEnv("FAKTORY_REDIS_TIMEOUT")
```
Comment (yakout) [inline storage/redis.go]: remove extra space

File: storage/redis.go [hunk 3]
```diff
@@ -131,8 +131,15 @@ func BootRedis(path string, sock string) (func(), error) {
 			}
 		}()
 	}
-
-	secs := 60
+	secs_env, isset := os.LookupEnv("FAKTORY_REDIS_TIMEOUT")
```
Comment (hossamElfar) [inline storage/redis.go]:
```suggestion
	timeoutSec, isSet := os.LookupEnv("FAKTORY_REDIS_TIMEOUT")
```

File: storage/redis.go [hunk 4]
```diff
@@ -131,8 +131,15 @@ func BootRedis(path string, sock string) (func(), error) {
...
+	secs := 120
```
Comment (hossamElfar) [inline storage/redis.go:135]:
```suggestion
	timeout := 120
```

Comment (phawzy) [reply storage/redis.go:135]: I dont think I need to change variable name that is already initialized in the code before and used later - I just added the feature

**Review by yakout — APPROVED** [review]

**Pull request merged**